models/issue.go

```diff
@@ -46,10 +46,13 @@ type Issue struct {
 	NumComments     int
 	Ref             string
 
-	DeadlineUnix util.TimeStamp `xorm:"INDEX"`
-	CreatedUnix  util.TimeStamp `xorm:"INDEX created"`
-	UpdatedUnix  util.TimeStamp `xorm:"INDEX updated"`
-	ClosedUnix   util.TimeStamp `xorm:"INDEX"`
+	DeadlineUnix   util.TimeStamp `xorm:"INDEX"`
+	DeadlineString string         `xorm:"-"`
```

Comment:
It seems this field is unnecessary.

Reply:
You could use something like `.DeadlineUnix.FormatShort`.

Reply:
Thanks, I didn't know that function exists.

On another note, what do you think about putting date formats in the corresponding locale files meaning we could have different date formats for every locale?